applovin_max/ios/Classes/AppLovinMAX.m

```diff
@@ -315,6 +315,138 @@ - (void)setTestDeviceAdvertisingIds:(NSArray<NSString *> *)testDeviceAdvertising
     }
 }
 
+#pragma mark - Data Passing
+
+- (void)setTargetingDataYearOfBirth:(nonnull NSNumber *)yearOfBirth
```

Comment:
We usually just assume `nonnull` and don't explicitly annotate it (otherwise it gets too verbose if we do it all over the codebase). `nullable` annotations are fine though.

Reply:
Ok, thanks.